announce v0-1-1

jldec opened the pull request.

jldec committed:
announce v0-1-1

trisberg commented on _posts/2018-08-09-announcing-riff-0-1-1-on-Knative.md:
````diff
@@ -0,0 +1,56 @@
+
+```sh
+# remove everything including istio without prompting 
+riff system uninstll --istio --force
````
'uninstll' should be 'uninstall'

jldec committed:
typo

scothis committed:
polish

- update whitespace
- clarify the 0.1.1 is the riff version, not Knative
- add link to release
- minor grammar fixes
